virt/virt-4-13-release-notes.adoc

```diff
@@ -117,6 +114,9 @@ Removed features are not supported in the current release.
 //CNV-19043 Release note: CHANGE Remove rhel6 support
 * Red Hat Enterprise Linux 6 is no longer supported on {VirtProductName}.
 
+//CNV-23499: Carry over/repeat removed feature from version 4.12 
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed for all new deployments. In {VirtProductName} 4.13, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. Only in the case of an upgrade will a previously installed legacy HPP custom resource still be supported.
```

Comment:
I feel like the "Only in the case of..." sentence is a bit wordy and awkward.
Maybe change it to "Legacy HPP custom resource is supported only if it had been installed on a previous version of {VirtProductName}."

Reply:
Igtm